src/Cookie.php

```diff
@@ -111,29 +104,32 @@ public function setOptions(array $options): void {
         $this->options = $options;
     }
 
-    public function getCookie(string $name): mixed {
-        if (isset($_COOKIE[$name])) {
-            return $_COOKIE[$name];
+	/**
+	 * @return mixed|null
```

Comment:
`mixed` typehint is not available in php 7.x. This is only available since version 8.0.